tests/e2e_tests/subcommands/stake/test_set_revoke_child_hotkeys.py

```diff
@@ -0,0 +1,260 @@
+import bittensor
+from bittensor.commands import (
+    RegisterCommand,
+    StakeCommand,
+    RegisterSubnetworkCommand,
+    SetChildCommand,
+    SetChildrenCommand,
+    RevokeChildCommand,
+    RevokeChildrenCommand,
+)
+from tests.e2e_tests.utils import setup_wallet
+
+"""
+Test the set child hotkey singular mechanism. 
+
+Verify that:
+* No children hotkeys at subnet creation
+* Subnet owner an set a child hotkey
+* Child hotkey is set properly with proportion
+* Subnet owner can revoke child hotkey
+* Child hotkey is properly removed from subnet
+"""
+
+
+def test_set_revoke_child(local_chain, capsys):
```

Comment:
Pls add at least single line docstring to the test case.

Reply:
There is a docstring above this test that explains what its function is. Do you want me to move it down?

Reply:
Yeah, that would be great. Otherwise it looks like a description for a module. Thank you